angular-portfolio/src/app/bio/bio.component.scss

```diff
@@ -0,0 +1,39 @@
+.bio {
+    flex: 50%;
+    margin-top: 5%;
+    text-align: center;
+}
+
+.flex-container {
+    display: flex;
+    flex-direction: row;
+    height: 75vh;
+}
+
+p {
+    font-size: 24px;
+    line-height: 150%;
+    padding-left: 40px;
+    padding-top: 80px;
+    padding-right: 150px;
+    text-align: left;
+}
+
+.picture {
+    flex: 50%;
+    width: 50vw;
+}
+
+.picture img {
+    border-radius: 50%;
+    width: 50%;
+    margin-left: 35%;
+    margin-top: 3vh;
+}
```

Comment:
This might be a good place to use nesting (it will help bundle the associated styles together):
https://www.w3schools.com/sass/sass_nesting.asp